topics/fair/tutorials/fair-clinical/tutorial.md

```diff
@@ -0,0 +1,283 @@
+[comment]: <> (to do)
+
+In the UK, Clinical Practice Research Datalink ([CPRD](https://cprd.com)) collects anonymised patient data from a network of GP practices links this data to a range of other health-related data and provides a longitudinal, representative UK population health dataset. 
+
+The [Health Data Research Innovation Gateway](https://www.healthdatagateway.org/) can help researchers to find and discover UK health datasets, tools and other resources to further health research.
+
+> <hands-on-title>Explore HDRUK Innovation Gateway for datasets - {% icon time %} 5 minutes</hands-on-title>
+> 
+> 1. Click on Datasets in the [Health Data Research Innovation Gateway](https://www.healthdatagateway.org/)
+> 
+> 2. In the filters, expand Publisher and in Search Filter type CPRD and enter  
+> 
+> 3. Select CPRD  
+> 
+> 4. Expand Phenotype and search for rheumatoid, and select Rheumatoid Arthritis  
+> 
+> 5. Take a look at the information about the two datasets  
+> 
+> 6. Clear the filters, and search for datasets related to your areas of interest  
+>
+{: .hands_on}
+
+# The importance of making clinical datasets FAIR
+
+## Associations between clinical features and omics
+In some recent work in rheumatoid arthritis [The RA-MAP Consortium, 2022](https://www.nature.com/articles/s41597-022-01264-y), it was demonstrated that simple patient demographics such as sex, ethnicity and age are all drivers of expression variation in addition to disease activity.  Principal component driver plots highlighted critical associations between diverse clinical features and omics.  This shows how rich clinical information may be key to analysis in some, if not, many diseases.  
+
+![RA-MAP.](../../images/RAMAP_graphic.png)
+
+> <hands-on-title>Do the datasets have all these characteristics - {% icon time %} 5 minutes</hands-on-title>
+>
+> For your domain, look at datasets on ArrayExpress and GEO and see how many of them have sex, age, ethnicity/race, and some measure of disease activity
+>
+{: .hands_on}
+
+## Powering up cohorts for analyses and machine learning
+For many diseases (or across a family of diseases), there is a need for well-powered cohorts with hundreds to thousands of participants, for analysis purposes.  FAIR principles have been widely promoted for omics datasets, but as shown, there are key challenges to meta-analysis.  This is illustrated by an example of collating 17 public rheumatoid arthritis studies.  The clinical data in most cases were sparse and incomplete.  For example, of the curated 17 studies, three did not include sex, 5 did not include age and only one study included ethnicity or race.  The limitations in the availability of clinical data substantially diminish the value of public data sets and on the surface to not adhere to FAIR principles.
+
+[comment]: <> (Is there a link to the rheumatoid arthritis study example?)
+[comment]: <> (need to get link from Mike)
+
```

Comment:
nice! hadn't seen anyone track their TODOs like this before but it's a nice idea :)